src/tests/electron/tests/android-setup-prompt-install-service.test.ts

```diff
@@ -35,25 +38,30 @@ describe('Android setup - prompt-install-service ', () => {
     });
 
     it('initial component state is correct', async () => {
+        await dialog.waitForDialogVisible('prompt-install-service');
         const [closeId, nextId] = [leftFooterButtonAutomationId, rightFooterButtonAutomationId];
         expect(await dialog.isEnabled(getAutomationIdSelector(closeId))).toBe(true);
         expect(await dialog.isEnabled(getAutomationIdSelector(nextId))).toBe(false);
         expect(await dialog.isEnabled(getAutomationIdSelector(installAutomationId))).toBe(true);
     });
 
     it('install button triggers installation, prompts for permission on success', async () => {
+        await dialog.waitForDialogVisible('prompt-install-service');
         await setupMockAdb(defaultDeviceConfig);
         await dialog.client.click(getAutomationIdSelector(installAutomationId));
         await dialog.waitForDialogVisible('prompt-grant-permissions');
     });
 
     it('install button triggers installation, prompts correctly on failure', async () => {
+        await dialog.waitForDialogVisible('prompt-install-service');
         await setupMockAdb(simulateServiceInstallationError(defaultDeviceConfig));
         await dialog.client.click(getAutomationIdSelector(installAutomationId));
         await dialog.waitForDialogVisible('prompt-install-failed');
     });
 
-    it('should pass accessibility validation in both contrast modes', async () => {
+    it('spinner & prompt dialogs pass accessibility validation in both contrast modes', async () => {
```

Comment:
Slowing all of the tests when we only need to slow this one feels awkward. What if we left the existing tests as written, then did the test on rescan, something like this??

await setupMockAdb(delayAllComands(2500, simulateServiceNotInstaled(defaultDeviceConfig));
await dialog.client.click(installAutomationId);
await app.openAndroidSetupView('detect-service');
await scanForAccessibilityIssuesInAllModes(app);
await diaog.waitForDialogVisible(prompt-install-service);
await scanForAccessibilityIssuesInAllModes(app);

Reply:
after discussing offline I will make a new test class for the spinner alone to avoid making this test class too complex / slower than it needs to be